src/lib/ManufacturerLogo.tsx

```diff
@@ -0,0 +1,24 @@
+import type { CommonManufacturerLogoProps } from './manufacturers/CommonManufacturerLogoProps.js'
+import { AiPhoneLogo } from './manufacturers/index.js'
+
+export const manufacturerLogos = {
+  ai_phone: AiPhoneLogo,
+} as const
+
+type ManufacturerName = keyof typeof manufacturerLogos
+
+interface ManufacturerLogoProps
+  extends Omit<CommonManufacturerLogoProps, 'wordmark'> {
```

Comment:
Why is wordmark omitted here?

Reply:
tbh, i think it's safe to remove `wordmark`. I was thinking it would be useful to have a wordmark vs square version, but in the end, i saw that square format is good enough and I haven't encountered a use case where i thought to myself that it'd be better if I had a wordmark version of the logo.